tests/routes/admin_login.test.js

```diff
@@ -0,0 +1,55 @@
+const test = require('tape');
+const request = require('supertest');
+const jwt = require('jsonwebtoken');
+const createApp = require('../../react-backend/app');
+const { getConfig } = require('../../config');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('POST /api/admin/login | password match database hash', t => {
+  const app = createApp(config);
+  const dbConnection = app.get('client:psql');
+
+  const token = jwt.sign(
+    { email: 'jinglis12@googlemail.com' },
+    process.env.JWT_SECRET
```

Comment:
This has been renamed, and in any case, you should be referring to the `config` object from line 7 instead of relying explicitly on any environment variables.

Same goes for every occurrence of this in the tests added.